tools/xcodeproj-shims/clang-stub.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -eu
```

Comment:
Maybe a few comments explaining what the stub does?

Reply:
done